src/test/java/org/broadinstitute/hellbender/utils/read/AlignmentUtilsUnitTest.java

```diff
@@ -141,21 +141,59 @@ private GATKRead makeReadForAlignedToRefTest(final String baseString) {
             badHap.setAlignmentStartHapwrtRef(hapStart);
 
             final int expectedPos = 10130740;
-            tests.add(new Object[]{read, badHap, refStart, expectedPos, goodCigar});
+            tests.add(new Object[]{read, badHap, badHap, refStart, expectedPos, goodCigar});
         }
 
+        // example where left-align generates a leading deletion
+        {
+            final String ref = "CTGAACGTAACCAAAATCAATATGGATACTGAGAAATACTATTTAATAAAGACATAAATTAGACTGCTAAAAAAAATTAAAGAAATTTCAAAAGAGAATCCACCTCTTTTCCTTGCCAGTGCTCAAAAGTGAGTGTGAATCTGGTGGCTGTGGGGCTGTTTTTGGTGTGGCTCTTTGGACCAGCCTGCCTGGTAATTCAAGCCTGCCTCTCATTTCTG";
+            final String hap = "CTGAACGTAACCAAAATCAATATGGATACTGAGAAATACTATTTAATAAAGACATAAATTAGACTGCTAAAAAAAATTAAAGAAATTTCAAAAGAGAATCCACCTCTTTTCCTTGCCAGTGCTCAAAAGTGAGTGTGAATCTGGTGGCTGCGGGGCTGTTTTTGGTGTGGCTCTTTGGACCAGCCTGCCTGGTAATTCAAGCCTGCCTCTCATTTCTG";
+            final String hapCigar = "218M";
+            final String readBases = "GCTGCTTTTGGTGTGGCTCTTT";
```

Comment:
It's very hard for a reader of this test code to eyeball where the read bases start in the reference -- could you add a comment?

Reply:
I've added the pairwise alignments to make eyeballing the alignments easier.